src/Illuminate/Support/Arr.php

```diff
@@ -220,6 +220,30 @@ public static function flatten($array, $depth = INF)
     }
 
     /**
+     * Flatten a multi-dimensional array into a single level
+     * keeping the keys formatted into dot notation.
+     *
+     * @param  array $array
+     * @param string $prefix
+     * @return array
+     * @internal param int $depth
+     */
+    public static function flatten_keeping_keys($array, $prefix = '')
```

Comment:
Use camel case here

Reply:
ah right sorry, pasted in the helper, which can be snake case, to here and forgot that thanks